html/changelog2016.html

```diff
@@ -1959,7 +1959,7 @@ <h3 class='author'>Infernus44 updated:</h3>
 		<li class='rscadd'>You can now use knives and swords for surgery. I don't recommend it tho...</li>
 		<li class='rscadd'>Napalm now uses atmos friendly fire. Works similar to incendiary grenades</li>
 		<li class='rscdel'>Phoron no longer releases volatile fuel into atmos</li>
-		<li class='bugfix'>Fixed a minor bug where mar40 wasn't spawned correctly for mercs</li>
+		<li class='bugfix'>Fixed a minor bug where ak47 wasn't spawned correctly for mercs</li>
```

Comment:
```suggestion
		<li class='bugfix'>Fixed a minor bug where mar40 wasn't spawned correctly for mercs</li>
```
No need to change the old changelog.